### ISSUE TITLE
Error unbound variable

### ISSUE THREAD

**Issue by aquada** [pratiktri/wordpress_borg_backup#1]
`./wp_borg_backup.sh: line 107: SUDO_USER: unbound variable `

Consistently when running script.

**Comment by pratiktri** [comment]
I have since moved on to [Ghost](https://ghost.org/) and have very little resource for testing wordpress for now.
Feel free to create a pull request.

**Comment by aquada** [comment]
Ok thanks. I actually managed to use borgmatic to backup all the hosted websites and the complete MariaDB. I run this all with WordOps now, WordPress and other websites. How is Ghost working for you?

**Comment by pratiktri** [comment]
Glad to know you found a solution. WordOps is very good for managing wordpress.
Ghost is much simpler - does not need a lot of work to keep working. Much better for personal websites.

**Comment by aquada** [comment]
Yes, I look at Ghost and was impressed. I'm using Grav at the moment and a few others on WordPress. I think I'll look again at Ghost. Grav is nice as it's a flat filesystem, no DB, but it has its own idiosyncrasies
